src/encoded/auth0.py

```diff
@@ -96,6 +93,40 @@ def forget(self, request):
         return []
 
 
+def signup(request):
+    """
+    Create new user.
+
+    :param request: Pyramid request object
+    """
+    domain = 'encode.auth0.com'
+    url = "https://{domain}/userinfo?access_token={access_token}".format(domain=domain, access_token=request.json['accessToken'])
+    user_data_request = requests.get(url)
+
+    if user_data_request.status_code != 200:
+        raise HTTPBadRequest(explanation='Could not get user data')
+
+    users = request.registry[COLLECTIONS]['user']
+    user_data = user_data_request.json()
+
+    if not user_data['email_verified']:
+        raise HTTPBadRequest(explanation='Unverified e-mail')
+
+    user_info = {
+        'email': user_data['email'],
+        'first_name': user_data.get('given_name', '') or user_data.get('first_name', '') or user_data.get('name', ''),
+        'last_name': user_data.get('family_name', '') or user_data.get('last_name', ''),
+        'groups': [],
+        'submits_for': [],
+        'viewing_groups': [],
+        'status': 'current',
+    }
+    user = create_item(users.type_info, request, user_info)
```

Comment:
What if there's already an account with the same email address?

Reply:
The user is passed through. I spoke with Idan about this and he said the behavior is expected. So, to be sure we are on the same page, If I am registered with "me@me.com" on linkedin, I can enter use "me@me.com" on google to enter the system if I have a google account. I seems strange to me too.

Reply:
My main concern with this code is that it could accidentally create a bunch of spam accounts. Should definitely try to break it in QA.